src/components/Provider.jsx

```diff
@@ -0,0 +1,94 @@
+import { nanoid } from "nanoid";
+import React, { useState, createContext } from "react";
+
+export const Context = createContext(null);
+
+const Provider = (props) => {
+    const loadLocalStorage = () => {
+        return window.localStorage.getItem("todos")
+            ? [...JSON.parse(window.localStorage.getItem("todos"))]
+            : [{ text: "initial todo", id: null, checkness: false }];
+    };
+    const updateLocalStorage = () => {
+        window.localStorage.setItem("todos", JSON.stringify(todos));
+    };
+    const [todos, setTodos] = useState(() => loadLocalStorage());
+
+    const [currentTodo, setCurrentTodo] = useState({
+        text: "",
+        id: null,
+        checkness: false,
+    });
+    const increaseTodos = (todo) => {
+        if (!todo) return;
+        if (todo.text.trim() === "") return;
+        setTodos([
+            ...todos,
+            { text: todo.text.trim(), id: nanoid(), checkness: false },
+        ]);
+        setCurrentTodo({ text: "", id: null });
```

Comment:
The object set for the current todo is missing the `checkness`, which might cause bugs.